src/systems/ec/implementations/react/components/gallery/src/Gallery.jsx

```diff
@@ -32,6 +35,20 @@ export const Gallery = ({
 
       <Button label={viewAllLabel} variant="ghost" data-ecl-gallery-all />
 
+      <div className="ecl-gallery__footer">
+        <Link
+          href="#"
+          aria-label={footerLinkLabel}
```

Comment:
the aria-label should be different from the link label. So it requires a dedicated parameter (already provided in the Link component)